docs/source/data-access-api/openapi.yml

```
@@ -143,7 +144,8 @@ paths:
             type: string
         - in: query
           name: version
-          description: The version of data product in the format ``vXX-YY`` (e.g. ``v01-01``).
+          description: The version of data product in the format ``vXX-YY`` (e.g. ``v01-01``). You can also choose to
```

```suggestion
          description: The version of data product in the format ``vNNN`` (e.g. ``v001``). You can also choose to
```
This documentation has become outdated and still references the old version number format. Could you update this?